src/app/form-error-messages/form-error-messages.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, Input } from '@angular/core';
+import { AbstractControl, AbstractControlDirective } from '@angular/forms';
+
+@Component({
+  selector: 'planet-form-error-messages',
+  template: `
+    <ul class="text-danger" *ngIf="shouldShowErrors()">
+      <li *ngFor="let error of listOfErrors()">{{error}}</li>
+    </ul>
+  `,
+  styles: [
+    `ul {
+      list-style-type: none;
+      font-size: .8rem;
+      padding-left: .2em;
+    }`
+  ]
+})
+export class FormErrorMessagesComponent {
+  private static readonly errorMessages = {
+    // even though they have the same return type, different names are used since it's more clear & easier to understand/debug
+    required: () => 'This field is required',
```

Comment:
is this component going to be used for other forms as well?
If so can we make some of these like required as well to be params as it seems like errors will show up accumulated in one place.